README.md

```diff
@@ -1,2 +1,100 @@
 # brigade-robot-framework
-This repository uses Azure/Brigade for running in cluster (Kubernetes) Robot Framework tests.
+
+This repository uses [Azure/Brigade](https://github.com/Azure/brigade) for running in cluster ([Kubernetes](https://kubernetes.io/)) Robot Framework tests.
+
+## Prerequisites
+
+1. Have a running [Kubernetes](https://kubernetes.io/docs/setup/) environment
+2. Setup [Helm](https://github.com/kubernetes/helm) - this assumes Helm on your Host regardless of the Helm container used later on.
+3. (optional) an NFS Provisioner - my environment uses Kubernetes clusters running on standalone bare metal machines. 
+    Recommended simple NFS Provisioner is [IlyaSemenov/nfs-provisioner-chart](https://github.com/IlyaSemenov/nfs-provisioner-chart).
+    Follow the instructions for adding the helm repo. At install set this as the default Provisioner:
+```bash
+$ helm install --name nfs-provisioner --namespace nfs-provisioner nfs-provisioner/nfs-provisioner --set defaultClass=true
+```
+If  not already set up on your host, add nfs-commons
+```bash
+$ apt-get install nfs-commons
+```
+
+## Install
+
+### Set up Brigade
+
+Follow the [quick-start guide](https://github.com/Azure/brigade#quickstart):
+
+Install Brigade into your Kubernetes cluster is to install it using Helm.
+
+```bash
+$ helm repo add brigade https://azure.github.io/brigade
+$ helm install -n brigade brigade/brigade
+```
+
+To manually run Brigade Projects the **brig** binary is required. Follow the
+[Developers Guide](https://github.com/Azure/brigade/blob/master/docs/topics/developers.md)
+to build the binary. Assuming Brigade is cloned and prerequisites met, simply run:
+```bash
+$ make brig
+```
+Test **brig** with `brig version`
+
+### Install brigade-robot-framework
+
+Clone brigade-robot-framework  and change directory
+```bash
+$ git clone https://github.com/charter-ctec/brigade-robot-framework
+$ cd brigade-robot-framework
+```
+Helm install brigade-robot-framework
+> note the name and namespace can be customized
+```bash
+$ helm install --name robot-brigade brigade/brigade-project -f robot-values.yaml \
+    --set secrets.dvc_user=<device_user> \
+    --set secrets.dvc_pass=<device_password> \
+    --set secrets.dvc_ipaddr=<device_ip_address> \
+    --set secrets.dvc_name=<device_name>
+```
+
+### Set up Minio
+
+```bash
+$ kubectl apply -f minio-deployment/minio-deployment.yaml
+```
+
+## Usage
+
+Manually run the project. The project name is the same as the project value in
+the *robot-values.yaml*
+```bash
+$ brig run -f brigade_robot.js charter-ctec/brigade-robot-framework
```

Comment:
if the `brigade_robot.js` in the repo is the code we want to run, we don't have to manually input a file here
`brig run charter-ctec/brigade-robot-framework` will run the `brigade.js` in the repo